README.md

```diff
@@ -69,6 +69,10 @@ If you have any questions, feel free to open a GitHub issue on a related project
 * Slack: Join the #WasmEdge channel on the [CNCF Slack](https://slack.cncf.io/)
 * Twitter: Follow @realwasmedge on [Twitter](https://twitter.com/realwasmedge)
 
+## Adopters
+
+Check out our [list of Adopters](https://wasmedge.org/docs/contribute/users/) who are using WasmEdge in their projects.
```

Comment:
Can we add one sentence to call for other people to add their name if they're using WasmEdge?

Reply:
Can you please review now

Reply:
Ok

Reply:
Sorry I may make you confused. The sentence you added is good. But I don't think we need two headlines for Adopter. Can you add the sentence behind `Check out our [list of Adopters](https://wasmedge.org/docs/contribute/users/) who are using WasmEdge in their projects.`?

Reply:
It is oky